src/features/Table/index.tsx

```diff
@@ -3,20 +3,44 @@
 // import SearchBar from '../SearchBar';
 import Row from './Row';
 
-const Table = (): JSX.Element => {
+export interface TableEmployee {
+  id: string | number
+  name: string
+  email: string
+  role: string
+};
+
+interface IProps {
+  employees: TableEmployee[]
+  onSelect?: (employee: TableEmployee) => void
+}
+
+const Table = ({
+  employees,
+  onSelect,
+}: IProps): JSX.Element => {
+  const isClickable = !(onSelect == null);
```

Comment:
Like I explained above , you should describe the reason of this kind of change.

Reply:
The way you wrote the code can lead to an excessive reload of the application because it will fire every time there is a change in the onSelect. It is a best practice to save variable in the state or to have it directly link to element of the DOM. 
Also, I'm not sure if this is good practice either `onSelect == null`. We could refactor it out later